.github/workflows/ci.yml

```diff
@@ -112,6 +112,20 @@ jobs:
           crate: dylint-link
           version: 2
 
+      - name: Download and run latest `substrate-contracts-node` binary for macOS
+        if: runner.os == 'macOS' 
+        run: |
+            curl -L -o substrate-contracts-node.zip 'https://gitlab.parity.io/parity/mirrors/substrate-contracts-node/-/jobs/artifacts/main/download?job=build-mac' && \
+            unzip substrate-contracts-node.zip && \
+            chmod +x artifacts/substrate-contracts-node-mac/substrate-contracts-node &&
+            ./artifacts/substrate-contracts-node-mac/substrate-contracts-node -linfo,runtime::contracts=debug  2>&1 | tee /tmp/contracts-node.log &
```

Comment:
For the ink! CI we use GitHub Actions (for Mac OS and Windows) and GitLab (for Linux). 

For the `cargo-contract` CI we have just GHA (for Mac OS, Windows and Linux). Hence you need to account for Linux here as well. Probably this code from the Mac step works on Linux too, if the Docker image has `unzip` installed.

Reply:
Yes I have noticed but it was a first test to check if at least it was working for Mac and WIndows but it doesnt

Reply:
Here's the Linux command: https://github.com/paritytech/ink-examples/blob/main/.github/workflows/ci.yml#L74-L80.